src/app/components/sources/source-note/source-note.component.html

```diff
@@ -0,0 +1 @@
+<evt-note [data]="dataForNote"></evt-note> 
```

Comment:
dataForNote is not the bast name. Can you change it in "data"?

Reply:
data (and _data) was already used in the component .ts